go.mod

```diff
@@ -27,13 +27,13 @@ require (
 	github.com/natefinch/lumberjack v2.0.0+incompatible
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.1
-	github.com/robertkrimen/otto v0.0.0-20211024170158-b87d35c0b86f
+	github.com/robertkrimen/otto v0.0.0-20221025135307-511d75fba9f8
```

Comment:
Is the otto change intentional in this PR?

Reply:
No, will revert and figure out where this came from.